include/linux/mod_devicetable.h

```diff
@@ -838,4 +838,12 @@ struct mhi_device_id {
 	kernel_ulong_t driver_data;
 };
 
+#define ANCILLARY_NAME_SIZE 20
+#define ANCILLARY_MODULE_PREFIX "ancillary:"
+
+struct ancillary_device_id {
+	char name[ANCILLARY_NAME_SIZE];
```

Comment:
name variable here is confusing specially when this is used as device_id.
name confuses with the actual device name.

so please name it as 
ANCILLARY_DEVICE_ID_SIZE
char device_id[ANCILLARY_DEVICE_ID_SIZE];

Reply:
@paravmellanox There are plenty of other instances in teh same file which use name in their device_is structs in cluding platform_device _id. I think name is more useful than device_id.

Reply:
device_id is typically used when you have a physical ID, e.g. vendor/part. There's nothing of this sort here.